### PR TITLE
(update) game score

### PR DESCRIPTION
**Pull request by BillyGoat12**

### CODE REVIEW

**Review by BenYoss — APPROVED** [review]

File: client/src/components/App.jsx [hunk 1]
```diff
@@ -5,6 +5,8 @@ import { Snackbar } from '@material-ui/core';
 import { Alert } from '@material-ui/lab';
 import { io } from 'socket.io-client';
 import { makeStyles } from '@material-ui/core/styles';
+import _ from 'underscore';
```
Comment (jstrzesz) [inline client/src/components/App.jsx:8]: man underscore, it's been a minute since I've seen that

File: client/src/components/App.jsx [hunk 2]
```diff
@@ -52,6 +55,18 @@ const App = () => {
   // function Alert(props) {
```
Comment (jstrzesz) [inline client/src/components/App.jsx]: remove if unnecessary, otherwise label with a `//TODO: tag`

File: client/src/components/App.jsx [hunk 3]
```diff
@@ -52,6 +55,18 @@ const App = () => {
...
   //   return <MuiAlert elevation={6} variant="filled" {...props} />;
   // }
+
+  useEffect(() => {
+    if (user) {
+      axios.get(`/data/deckCards/${user.default_deck}`)
+        .then(({ data }) => {
```
Comment (jstrzesz) [inline client/src/components/App.jsx:59]: need a .catch block

File: client/src/components/GameEnv/2DEnv/2DEnv.jsx [hunk 4]
```diff
@@ -55,6 +55,7 @@ const TwoDEnv = ({
     setTimeout(() => {
       if (!turn && !user.id_enemy && user) {
         let enmeyResource = resourceCount;
+        console.info(enmeyResource);
```
Comment (jstrzesz) [inline client/src/components/GameEnv/2DEnv/2DEnv.jsx]: remove if not needed

File: client/src/components/GameEnv/GameEnv.jsx [hunk 5]
```diff
@@ -42,8 +43,16 @@ const GameEnv = ({ setNav }) => {
     });
   }
   if ((HP <= 0 || enemyHP <= 0) && !done) {
+    if (enemyHP <= 0 && HP <= 0) {
+      axios.get(`data/wins/${user.id}/${user.total_win + 0.5}`);
```
Comment (jstrzesz) [inline client/src/components/GameEnv/GameEnv.jsx:47]: do these need to have .thens attached to them?

**Review by jstrzesz — CHANGES_REQUESTED** [review]
see comments

File: client/src/components/GameEnv/GameEnv.jsx (hunk 5 above)
Comment (BillyGoat12) [reply client/src/components/GameEnv/GameEnv.jsx:47]: not really

Comment (BillyGoat12) [reply client/src/components/GameEnv/GameEnv.jsx:47]: should i?

Comment (jstrzesz) [reply client/src/components/GameEnv/GameEnv.jsx:47]: if it works as intended then its fine

**Review by agrem28 — APPROVED** [review]

**Pull request merged**